ui/desktop/electron-app/src/helpers/spawn-promise.js

```diff
@@ -78,7 +78,25 @@ module.exports = {
    */
   spawnSync(command) {
     const childProcess = spawnSync(path(), command);
-    const rawOutput = childProcess.output.toString();
+    const rawOutput = childProcess.stdout.toString();
```

Comment:
why are we changing this to stdout?

Reply:
I think this was an oversight when originally written. When you call `toString` on output, you're getting all the io streams (stdin, stdout, stderr) which I doubt was the intention. This causes problems if you're expecting just normal output from the stream to parse into JSON since you would have to clean up the other output/input.

I don't think it was ever an issue because we used this previously only for checking the boundary version which we used regex for and never parsed it. (I'm not sure why we didn't just format it as json? Perhaps it didn't have it at the time? 🤷 )